README.md

```diff
@@ -176,6 +176,7 @@ Install Velero, including all prerequisites, into the cluster and start the depl
 ```bash
 velero install \
     --provider azure \
+    --plugins velero/velero-plugin-for-microsoft-azure:v1.0.0 \
```

Comment:
Let's set the version/tag to v1.0.0-beta.1 for now. We'll have to update again when we do the GA release.